### ISSUE TITLE
AEWeb: Suggestion to be able to see old version

### ISSUE THREAD

**Issue by bchamagne** [archethic-foundation/archethic-node#1046]
Currently AEWeb will always fetch the latest transaction of the chain in order to display the latest version of the website.
I would like to alter this behaviour: 

1. If URL is on a genesis address: fetch latest transaction and then display the website described in the transaction 
2. If URL is not a genesis address: display the website described in the transaction

I think it's really good for transparency. And it allows people playing with the explorer to actually see what there is behind a transaction.

**Comment by samuelmanzanera** [comment]
Yes, it can be an interesting approach. Either using parameters or using the model you described.
But with the mentioned behavior, it will be simplified the API and the usage regarding the parameter approach.
